CodeU_Assignement_3/src/codeu_assignement_3/KemWordSearch.java

```diff
@@ -1,16 +1,138 @@
 package codeu_assignement_3;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+
 /**
  *
  * @author Karine
  */
 public class KemWordSearch 
 {
-    private KemDictionary m_dictionary;
+    /*
+    Input: the number of rows, number of columns, 
+           a 2-dimensional array of characters (of the native char data type)
+           a dictionary
+    
+    Process: 
     
-    public KemWordSearch (KemDictionary dictionary) 
+    Output: return the set of all-words found
+    */
+    String[] wordSearch(int row, int col, char[][] grid, KemDictionary dictionary)

```

Comment:
It was the input in the interface in the question sent in slack, I just followed the interface...
 Saying this:
"Your function receives the number of rows, number of columns, a 2-dimensional array ofcharacters (of the native char data type), and the dictionary. You should return the set of allwords found."